public/event.less

```diff
@@ -140,7 +140,7 @@
     padding: 0;
   }
 
-  .plugin-calendar-event-editor-dates {
+  .plugin-calendar-event-editor-dates .plugin-calendar-event-editor-date {

```

Comment:
What does this change make it look like?

Reply:
I'm asking for a a screenshot

Reply:
This is how it looks before the change:
![before](https://user-images.githubusercontent.com/6916385/30525119-a422941a-9c00-11e7-9a58-565e7f6d39f0.JPG)

And this is after the change:
![after](https://user-images.githubusercontent.com/6916385/30525118-a41ec772-9c00-11e7-9039-9b82d43872eb.JPG)

Reply:
That's what I was fearing. I want it to be side-by-side on desktop.